database/cassandra/cassandra.go

```diff
@@ -59,11 +61,24 @@ func (p *Cassandra) Open(url string) (database.Driver, error) {
 		MigrationsTable: migrationsTable,
 	}
 
+	var username, password string
+
+	if u.User != nil {
+		username = u.User.Username()
+		password, _ = u.User.Password()
+	} else {
+		username = "cassandra"
```

Comment:
where is this default coming from?

Reply:
It's the default username/password when enabling `PasswordAuthentication` authenticator in Cassandra. See https://docs.datastax.com/en/cassandra/2.1/cassandra/security/security_config_native_authenticate_t.html.